### PR TITLE
feat: Instinct documentation link in ROCm docs

### PR DESCRIPTION
**Pull request by saadrahim**

### CODE REVIEW

**Comment by peterjunpark** [comment]
@saadrahim Should the Instinct/dcgpu site be added to the ROCm theme header?

image

**Comment by saadrahim** [comment]
> @saadrahim Should the Instinct/dcgpu site be added to the ROCm theme header?
> 
> image

Please see the preview. I've incorporated  your change. The link to InfinityHub is dropped in favor of Instinct docs.

**Review by alexxu-amd — APPROVED** [review]
Looks good

**Review by lpaoletti — CHANGES_REQUESTED** [review]
We want to discuss the changes to the ROCm docs interface before any changes are made.

File: src/rocm_docs/rocm_docs_theme/flavors/instinct/header.jinja [hunk 1]
```diff
@@ -6,7 +6,7 @@
     {% else %}
         {% set version_name = branch %}
     {% endif %}
-    <a class="klavika-font hover-opacity" href="{{ projects['rocm'] }}">Instinct&#8482; Hub</a>
+    <a class="klavika-font hover-opacity" href="{{ projects['rocm'] }}">Instinct&#8482; Documentation</a>
```
Comment (lpaoletti) [inline src/rocm_docs/rocm_docs_theme/flavors/instinct/header.jinja]: Suggest removing TM symbol - shouldn't be in the Header

File: src/rocm_docs/rocm_docs_theme/flavors/rocm/header.jinja [hunk 2]
```diff
@@ -26,7 +26,7 @@ set nav_secondary_items = {
     "GitHub": repo_url,
     "Community": "https://github.com/ROCm/ROCm/discussions",
     "Blogs": "https://rocm.blogs.amd.com/",
-    "Infinity Hub": "https://www.amd.com/en/developer/resources/infinity-hub.html",
+    "Instinct&#8482; Docs": "https://dcgpu.docs.amd.com",
```
Comment (lpaoletti) [inline src/rocm_docs/rocm_docs_theme/flavors/rocm/header.jinja:29]: Docs wants to discuss this.

File: src/rocm_docs/rocm_docs_theme/flavors/instinct/header.jinja (hunk 1 above)
Comment (saadrahim) [reply src/rocm_docs/rocm_docs_theme/flavors/instinct/header.jinja]: TM is already in the ROCm docs header.
image

**Review by alexxu-amd — APPROVED** [review]

**Review by lpaoletti — APPROVED** [review]

**Pull request merged**